packages/suite-desktop-ui/src/MainDesktop.tsx

```diff
@@ -112,7 +112,10 @@ export const init = async (container: HTMLElement) => {
         });
     }
 
-    const loadModules = await desktopApi.loadModules(null);
+    console.log('store.getState()', store.getState());
+    const loadModules = await desktopApi.loadModules({
+        legacyBioAuthEnabled: store.getState().bioAuth.bioAuthEnabled,
```

Comment:
store.getState() returns `any` so this is a must have..
```suggestion
        legacyBioAuthEnabled: store.getState()?.bioAuth?.bioAuthEnabled
```